src/components/service_node/service_node_staking.vue

```diff
@@ -170,6 +232,38 @@ export default {
       this.service_node.key = key;
       this.service_node.amount = minContribution;
     },
+    minStake() {
+      const node = this.getNodeWithPubKey();
+      return this.getMinContribution(node);
+    },
+    maxStake() {
+      const node = this.getNodeWithPubKey();
+      return this.openForContributionLoki(node);
+    },
+    getFeeDecimal(node) {
+      const operatorPortion = node.portions_for_operator;
+      return (operatorPortion / 18446744073709551612) * 100;
+    },
+    buildDialogFieldsSweep(txData) {
+      this.buildDialogFields(txData);
+    },
+    getNodeWithPubKey() {
+      const key = this.service_node.key;
+      const nodeOfKey = this.awaiting_service_nodes.find(
+        n => n.service_node_pubkey === key
+      );
+      let node = nodeOfKey ? nodeOfKey : false;
+      if (!node) {
```

Comment:
you should be able to use `nodeOfKey` here directly
```suggestion
      if (!nodeOfKey) {
```